packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4193,6 +4198,16 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlag.isChecked, value!);
   }
 
+  // TODO(schectman): Uncomment once engine PR goes through
```

Comment:
Style guide ([avoid abandonware](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abandonware)) says:
> For example, we delete rather than commenting out code. Commented-out code will bitrot too fast to be useful, and will confuse people maintaining the code.
Although, you do plan on following up on this in a PR afterward...

Reply:
Could you just take the cherry-pick the commented code and put into a separate patch as non-commented code. Then you can just land that one after your engine PR lands.